conf/messages

```diff
@@ -48,6 +48,7 @@ account.noneDue.return.subHeading = You have no returns due.
 account.payments.card.title = Payments
 account.payments.subHeading = You have a payment of {0} due on {1} for {2} to {3}.
 account.payments.makePayment = Make an ECL payment
+account.payments.setDirectDebit = Set up or amend a Direct Debit
```

Comment:
you can remove this - we no longer show 'Set up or amend a Direct Debit' as a link in our service

Reply:
Done.